release.sh

```diff
@@ -8,6 +8,6 @@ if [ -z "$PYTHON_BIN_PATH" ]; then
 fi
 
 CONFIGURE_DIR=$(dirname "$0")
-"$PYTHON_BIN_PATH" "${CONFIGURE_DIR}/script/release.py" "$@"
+"$PYTHON_BIN_PATH" -m "${CONFIGURE_DIR}/script/release.py" "$@"
```

Comment:
In my understanding, `-m` has no meaning.
Module name needs to follow after `-m`. (e.g. pip, pytest)

https://docs.python.org/3/using/cmdline.html

Reply:
thanks. yeah, it's true